### DISCUSSION TITLE
http_scrape query

### DISCUSSION THREAD

**Discussion by mans2singh** [vectordotdev/vector#14583] [Q&A] [answered]
Hi:

I am trying to use http_scape and wanted to find out if it supports template for query parameters.    My use case is that I need to pass start and end time parameters to the http request.

I have checked the [documentation for http_scrape](https://master.vector.dev/docs/reference/configuration/sources/http_scrape/#query) but did not see this option.  

If I have missed the docs, please let me know if it is possible to template the params.

Thanks

**Comment by spencergilbert** [comment, marked as answer]
👋 
As is, the upcoming `http_scrape` source doesn't support templating for the `query` options.

**Reply by mans2singh** [reply]
Thanks @spencergilbert.  Is there a possibility to support it in the future ?

**Reply by spencergilbert** [reply]
I can't think of a technical reason right now that would block that enhancement - it was likely just not part of the original requirements. We can look at prioritizing that feature enhancement based on community interest if you'd like to open an issue for it.

**Reply by mans2singh** [reply]
Thanks @spencergilbert
